test/integration/visitor_can_view_items_test.rb

```diff
@@ -14,4 +14,33 @@ class VisitorCanViewItemsTest < ActionDispatch::IntegrationTest
       assert page.has_content?("Dang Coconut")
     end
   end
+
+  test "visitor can view items by category" do
+
+    category_1 = OilType.create(name: "Lard")
+    category_2 = OilType.create(name: "Coconut Oil")
+
+    item_1 = Chip.create(name: "Slotachips", price: 20, description: "Super yummy", oil_type_id: category_1.id)
```

Comment:
Useless assignment to variable - `item_1`. Did you mean `item_2`?<br>Line is too long. [111/80]